state_machine/applications/flight/Tasks/log.py

```diff
@@ -0,0 +1,61 @@
+from lib.template_task import Task
```

Comment:
Name this file `log_task.py`

Reply:
I don't really like the convention of calling everything in the tasks folder a task, it seems redundant.
`Tasks.log_task` vs `Tasks.log` doesn't seem all that different to me